frontend/packages/dev-console/src/components/import/CreateSecretModal.tsx

```diff
@@ -21,15 +22,16 @@ type Props = CreateSecretModalProps & ModalComponentProps;
 const getSecretForm = (type: SecretTypeAbstraction) => {
   switch (type) {
     case SecretTypeAbstraction.source:
-      return withSecretForm(SourceSecretForm, true);
+      return withSecretForm(withTranslation()(SourceSecretForm), true);
```

Comment:
The components should be exported with translation support built in and not require the consumers to wrap them.